### DISCUSSION TITLE
How to ignore babel config file in root

### DISCUSSION THREAD

**Discussion by Murderlon** [parcel-bundler/parcel#7365] [Q&A] [answered]
Hi,

I would like to know if it's possible to ignore the babel config file in root? 

My use case: create small apps in a yarn workspace in the [Uppy](https://github.com/transloadit/uppy) monorepo and write e2e tests against them.

We have a babel config in our root, which we need for other reasons, but parcel automatically picks it up from the root even if I run it as a workspace (`yarn workspace cypress parcel parcel/index.html`)

The parcel build will fail with the auto detected babel config file so I need to ignore that.

**Comment by nickcernis** [comment, marked as answer]
I used the documentation here that explains how to ignore Babel so that Parcel 2 uses its own transpiler:

https://parceljs.org/languages/javascript/#usage-with-other-tools

In short, you have to add a `.parcelrc` to your project root that excludes the usual `@parcel/transformer-babel` from the transformers list:

```
{
  "extends": "@parcel/config-default",
  "transformers": {
    "*.{js,mjs,jsx,cjs,ts,tsx}": [
      "@parcel/transformer-js",
      "@parcel/transformer-react-refresh-wrap"
    ]
  }
}
```

(In config-default `@parcel/transformer-babel` normally appears at the top of that transformers list, so the override above ensures that Parcel excludes the Babel transformer and will not process your Babel config.) You can still use Babel for other tooling such as Jest.

**Reply by Murderlon** [reply]
This works, thanks!
